packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx

```diff
@@ -64,6 +67,10 @@ export class MeasureToolsUiItemsProvider implements UiItemsProvider {
       }
 
       if (toolbarOrientation === ToolbarOrientation.Vertical) {
+        // don't display measurement tools on a sheet view
+        const vp = IModelApp.viewManager.selectedView;
```

Comment:
Don't add handling here. The tool can be added regardless of the current view its just the tool will hide itself if the current view is a sheet, which could change without this code here rerunning

Reply:
where would I add it then?